receiver/filelogreceiver/README.md

```diff
@@ -17,37 +17,37 @@ Tails and parses logs from files.
+| `storage`                           | none                                 | The ID of a storage extension to be used to store file checkpoints. File checkpoints allow the receiver to pick up where it left off in the case of a collector restart. If no storage extension is used, the receiver will manage checkpoints in memory only.                                     |
+| `header`                            | nil                                  | Specifies options for parsing header metadata. Requires that the `filelog.allowHeaderMetadataParsing` feature gate is enabled. See below for details. This option causes an error if it is used when `start_at` is set to "end".                                                                   |
+| `header.pattern`                    | required for header metadata parsing | A regex that matches every header line.                                                                                                                                                                                                                                                            |
+| `header.metadata_operators`         | required for header metadata parsing | A list of operators used to parse metadata from the header.                                                                                                                                                                                                                                        |
+| `retry_on_failure.enabled`          | `false`                              | If `true`, the receiver will pause reading a file and attempt to resend the current batch of logs if it encounters an error from downstream components.                                                                                                                                            |
+| `retry_on_failure.initial_interval` | `1s`                                 | Time to wait after the first failure before retrying. The duration must contain the unit of time (e.g.: 200ms, 1s, 1m.) otherwise an error will be caused.                                                                                                                                         |
+| `retry_on_failure.max_interval`     | `30s`                                | Upper bound on retry backoff interval. Once this value is reached the delay between consecutive retries will remain constant at the specified value. The duration must contain the unit of time (e.g.: 200ms, 1s, 1m.) otherwise an error will be caused.                                          |
+| `retry_on_failure.max_elapsed_time` | `5m`                                 | Maximum amount of time (including retries) spent trying to send a logs batch to a downstream consumer. Once this value is reached, the data is discarded. Retrying never stops if set to `0`. The duration must contain the unit of time (e.g.: 200ms, 1s, 1m.) otherwise an error will be caused. |
```

Comment:
We don't need to say "otherwise an error will be caused".

Generally speaking, if we need to describe the same thing multiple times, we should put the description in a dedicated section and link to it.

Reply:
Just to clarify, for this statement "The duration must contain the unit of time. e.g.: `200ms`, `1s`, `1m`.", do you propose that I make a dedicated section for this since it's repeated multiple times?

Reply:
Yes, I think that's preferable to defining the format many times.

Reply:
I added a new section that describes that all configurations involving time need to have the unit of time specified. Additionally, I added links to that section (used an anchor to do so) in all configurations involving time. I did the same thing for other configurations (specifically for multiline config and supported encodings), please let me know if I should change anything.